sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0 (Unreleased)
+## 1.0.3 (Unreleased)
```

Comment:
If you are going to release this as soon as it's fixed, you could add the release date in this PR as well. Else, you will need to open a new one as the release pipeline will fail if there is no date specified in the CHANGELOG.

Reply:
we decided to introduce the release changes(version and release date changes) in separate PR dedicated to the release, thanks

Reply:
Ok. Reverted.